dandi/publish/views/draft_version.py

```diff
@@ -83,3 +88,36 @@ def draft_publish_view(request, dandiset__pk):
     dandiset.draft_version.lock(request.user)
     publish_version.delay(dandiset.id, request.user.id)
     return Response(status=status.HTTP_204_NO_CONTENT)
+
+
+@swagger_auto_schema(
+    method='POST',
+    request_body=UserSerializer(many=True),
+)
+@api_view(['POST'])
+@permission_required_or_403('owner', (DraftVersion, 'dandiset__pk', 'dandiset__pk'))
+@permission_classes([IsAuthenticatedOrReadOnly])
+def draft_owners_view(request, dandiset__pk):
+    draft_version = get_object_or_404(Dandiset, pk=dandiset__pk).draft_version
```

Comment:
It seems unforunate that we can't get `draft_version` automatically as a parameter, but I realize we're already in quite some debt with the nested routing. :shrug:

Maybe this is worth filing a bug over.